pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -25,27 +27,6 @@ export type pokemonDisplayObj = {
   };
 };
 
-export enum ElementType {
```

Comment:
This type is needed outside of the `PokemonDTO`, so it's been moved to an external constants file.